### PR TITLE
Bug 1199461 - Change "Carrier" and "Network Operator" to "Operator". R=gasolin

### PR DESCRIPTION
**Pull request by louisom**
Bug 1199461 - Change "Carrier" and "Network Operator" to "Operator" in `Settings app > Cellular & Data` panel. R=gasolin

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5d63ebee4f3ef6f05c4d39e32c32a217e8d2db45

File: apps/settings/locales/settings.en-US.properties [hunk 1]
```diff
@@ -325,11 +325,11 @@ cellularAndData=Cellular & Data
 # If you need to adapt your localization to the reduced space available
 # in headers, you can use a shorter string here.
 cellularAndData-header={{cellularAndData}}
-dataNetwork=Carrier
+dataNetwork=Operator
```
Comment (gasolin) [inline apps/settings/locales/settings.en-US.properties]: for strings we have to rename the identifer `dataNetwork` to other, so the tool can scan the difference and provide the changed strings to l10n team. So I suggest you to change `dataNetwork` to `dataOperator` and update the related html file.

File: apps/settings/locales/settings.en-US.properties [hunk 2]
```diff
@@ -325,11 +325,11 @@ cellularAndData=Cellular & Data
...
 dataConnectivity=Data
 dataConnection=Data Connection
 dataRoaming=Data Roaming
-networkOperator=Network Operator
+networkOperator=Operator
```
Comment (gasolin) [inline apps/settings/locales/settings.en-US.properties]: same here, suggest `netOperator` instead of `networkOperator`

Comment (grapherd) [inline apps/settings/locales/settings.en-US.properties]: Hi gasolin,
So I also need to change l10n identifer in html right?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5b2f52ae45cfda6e76bf849ac26d55c69f940084

File: apps/settings/locales/settings.en-US.properties [hunk 3]
```diff
@@ -325,11 +325,11 @@ cellularAndData=Cellular & Data
 # If you need to adapt your localization to the reduced space available
 # in headers, you can use a shorter string here.
 cellularAndData-header={{cellularAndData}}
-dataNetwork=Carrier
+dataOperator=Operator
 dataConnectivity=Data
 dataConnection=Data Connection
 dataRoaming=Data Roaming
-networkOperator=Network Operator
+netOperator=Operator
```
Comment (gasolin) [inline apps/settings/locales/settings.en-US.properties]: there's a string that use networkOperator
`networkOperator-header={{networkOperator}}`

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=965bfaf05025d37b6e45f8466db0abfcf00b9bd1

**Comment by gasolin** [comment]
need add `netOperator-header` in `carrier_detail.html` and `operator_settings.html`.
It's hard to find when you don't have device on hand. So its fine to me, I'll help you add the rest at  https://github.com/mozilla-b2g/gaia/pull/32040

**Pull request closed without merging**